.mergify.yml

```diff
@@ -0,0 +1,13 @@
+pull_request_rules:
+  - name: Automatic strict merge
+    conditions:
+      - base=master
+      - assignee=box-merge
```

Comment:
This is the explicit action to give the go ahead for mergify to do the merge right? Assigning the box-merge user?

Reply:
Yeah, that's the suggestion I'm making, anyway. Said user account doesn't exist yet, though, so I'm open to alternatives.

Reply:
In the absence of that user, we could use a `ok-to-merge` label as the go ahead